### ISSUE TITLE
New Section: Deploy to Production

### ISSUE THREAD

**Issue by endigo9740** [skeletonlabs/skeleton#1239]
### Link to the Page

https://www.skeleton.dev/docs/deployment (not yet created)

### Describe the Issue

As discussed with @Sarenor, I think we need a new Docs > Resources > Deployment section to pair with Quickstart. Quick start shows the initial setup, this Deployment sections shows you everything to do to prep for release.

This could include, but is not limited to the following:

1. Document @AdrianGonz97's PurgeCSS tool (when available)
2. Recommended tips for preparing for release - ensure your app is optimized, lighthouse stuff, etc.
3. Written + Video guides for deploying to the top provides (Vercel, Netlify, Amplify, etc)

To be clear - these will NOT be guide we write and maintain, but rather resources we can find online. Be that the official documentation for each service, or useful video guides like this one by Joy of Code:

https://www.youtube.com/watch?v=uAF4Yd-gddo

The primary message should be that there's not really anything special about deploying a Skeleton app compared to a typical SvelteKit app - aside from the upcoming PurgeCSS instructions for better optimization.

### Are you able to create a Pull Request with the fix?

Yes

**Comment by endigo9740** [comment]
Note: Vercel's official instruction uses their dedicated adapter, but I'm not sure we want to recommend that or not:
https://vercel.com/docs/frameworks/sveltekit

**Comment by LukeHagar** [comment]
I have deployed Skeleton apps numerous times for work projects, 

I have experience with 
Express/Node Adapter/Docker
Node Adapter/Heroku
Vercel

Happy to help contribute tips and gotchas there

**Comment by FarhanAliRaza** [comment]
Hello, I want to help with this.

**Comment by endigo9740** [comment]
Hey @FarhanAliRaza this is issue is already assigned. We're waiting on some pre-requisites before we push forward on it.

If you would like to volunteer on another issue, look for one that's not already assigned and has the "help wanted" tag.

**Comment by Sarenor** [comment]
And if you've got links to online resources we can share on the "Deploy to production" page that you think are a good fit, feel free to leave them here :)

**Comment by endigo9740** [comment]
With the launch of v2 we now have a dedicated section for the PurgeCSS tool here:
https://www.skeleton.dev/docs/purgecss

I'm leaning more and more towards deployment to set services (Vercel, Netlify, etc) fall under the jurisdiction on SvelteKit and their adapter system, so should aim to forward anyone interested in this topic here:
https://kit.svelte.dev/docs/adapters

Beyond that, I do welcome contributed blog posts on any other topics in this vicinity.
